funcbench/env.go

```diff
@@ -98,14 +99,16 @@ type GitHub struct {
 }
 
 func newGitHubEnv(ctx context.Context, e environment, owner, repo string, prNumber int) (Environment, error) {
-	if err := os.Chdir(os.Getenv("GITHUB_WORKSPACE")); err != nil {
+	rpath := os.Getenv("GITHUB_WORKSPACE")
```

Comment:
Can we change this env var name to something more appropriate since we'll be using Github Actions just to launch funcbench and not to actually run it?

Reply:
Will rename to `WORKSPACE`, and change to:
1. try to read from cli flag `workspace` if set
2. try to read env `WORKSPACE` if set
3. try to use current directory

I think at some point we will merge with local mode 😈